docs/spring-boot/app-starter.md

```diff
@@ -0,0 +1,145 @@
+# BDK Extension App Spring Boot Starter
+The Symphony BDK for Java provides a _Starter_ module that aims to ease extension app backend developments within a
+[Spring Boot](https://spring.io/projects/spring-boot) application.
+ 
+## Features
+- Configure extension app through `application.yaml`
+- Expose a REST API to perform [Circle of Trust authentication](https://developers.symphony.com/extension/docs/application-authentication).
+
+## Installation
+
+The following listing shows the `pom.xml` file that has to be created when using Maven:
```

Comment:
isn't this part going to be redundant with the generator?

Reply:
I think it will be useful when user wants to create project not using the generator